bin/MadGraph5_aMCatNLO/cards/production/13TeV/tt0123j_5f_ckm_LO_MLM/tt0123j_5f_ckm_LO_MLM_proc_card.dat

```diff
@@ -0,0 +1,14 @@
+import model sm-ckm_lepton_masses_no_b_mass
```

Comment:
Massive leptons are no longer needed.  (Pythia 8.205 finally handles massless leptons properly)
So you can just use
 import model sm-ckm_no_b_mass
(and this will also speed up madspin)